sdk/attestation/azure-security-attestation/samples/README.md

```diff
@@ -129,7 +128,7 @@ rules for attestation operations.
 
 The following is an example of an attestation policy document for an SGX enclave:
 
-```
+```text
```

Comment:
Should step 1 of setup include `--pre` in the pip command after this GA's, or rather be:

1. Install the Azure Attestation client library for Python with [pip](https://pypi.org/project/pip/):

```bash
pip install azure-security-attestation
```